_includes/md/mdhost.php

```diff
@@ -0,0 +1,67 @@
+<?php
```

Comment:
Can we make this a _common file between keyman.com and help.keyman.com please? (and will end up on api.keyman.com, but that's fine, it just won't be used).